awesomeCreateApp.js

```diff
@@ -15,8 +15,12 @@ const terminalOpts = {
 }
 
 const createApp = folderName => {
-  spawn('create-react-app', [folderName], terminalOpts)
+  checkPackageInstalled('create-react-app')
+  .then(() => 
+    spawn('create-react-app', [folderName], terminalOpts)
     .then( () => showQuestions(folderName))
+    //TODO ask for install the package
+  ).catch(() => console.log(`${emoji.get('scream_cat')} create-react-app is not installed`))
```

Comment:
I think we could improve this message, maybe it's better explain for the user how the `create-react-app` could be installed. Also, could you add some colors (Using chalk)? for example:

```
${emoji.get('scream_cat')} ${chalk.red('Error: ')} create-react-app is not installed
Install create-react-app using: npm i -g create-react-app
```

Reply:
Is it the time to use `showCreateAppError()` ?